esrally/utils/net.py

```diff
@@ -207,6 +210,18 @@ def download_http(url, local_path, expected_size_in_bytes=None, progress_indicat
         return expected_size_in_bytes
 
 
+def download_http(url, local_path, expected_size_in_bytes=None, progress_indicator=None):
+    logger = logging.getLogger(__name__)
+    for i in range(HTTP_DOWNLOAD_RETRIES + 1):
+        try:
+            return _download_http(url, local_path, expected_size_in_bytes, progress_indicator)
+        except urllib3.exceptions.ProtocolError as exc:
+            if i == HTTP_DOWNLOAD_RETRIES:
+                raise
+            logger.warning("Retrying after %s", exc)
```

Comment:
Should we have a delay in between retries? Alternatively we could consider using something like https://pypi.org/project/tenacity/?

Reply:
I considered introducing a new dependency but thought this was not needed. I did not introduce a delay because I only saw connectivity issues (connection reset, incomplete reads) and not because the server was overloaded. Maybe a simple 1 second sleep could be enough?

Reply:
Agreed on a simple sleep; I'd go for `5s`, but up to you.

Reply:
Done, please take another look.